### PR TITLE
[FSPE-5902] Create Accessibility documentation

### PR DESCRIPTION
**Pull request by jllr**

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://72cf8625ab8920745f01b01db26ce7f233905f63-chi-pr.enterprise.ctl.io" target="_blank"> https://72cf8625ab8920745f01b01db26ce7f233905f63-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1286/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://5aee08fa81759195ea36b65c2e84626a01a3b3d4-chi-pr.enterprise.ctl.io" target="_blank"> https://5aee08fa81759195ea36b65c2e84626a01a3b3d4-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1291/ for more details.

**Review by mattnickles — COMMENTED** [review]
@jllr Are you still working on organizing the Accessibility nav items into a third tier under Foundations?

**Comment by jllr** [comment]
@mattnickles I am not working on that as we had to talk about it, I sent the first PR with the documentation only. Let's discuss it later.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://4204eee64a48f4198006c4d8db5d7d47201b667b-chi-pr.enterprise.ctl.io" target="_blank"> https://4204eee64a48f4198006c4d8db5d7d47201b667b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1632/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://d693a5528eed5dfe77ce619f11931f7847c3a2c1-chi-pr.enterprise.ctl.io" target="_blank"> https://d693a5528eed5dfe77ce619f11931f7847c3a2c1-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1634/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cdf18a40896673292562530a9479aa509cff3979-chi-pr.enterprise.ctl.io" target="_blank"> https://cdf18a40896673292562530a9479aa509cff3979-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1652/ for more details.

File: src/website/layouts/partials/sidenav.pug [hunk 1]
```diff
@@ -11,6 +11,34 @@ ul.chi-tabs.-vertical.-lg.-animated
                 if section.disabled
                   //- li
                   //-   a(href=`${rootPath}${section.path}`, disabled="disabled") #{section.title}
+                else if section.accessibilityAccordion
+                  li
+                    .chi-accordion.accessibility-accordion
+                      .chi-accordion__item.-expanded
+                        button.chi-accordion__trigger
+                          .chi-accordion__title #{section.title}
+                          i.chi-icon.icon-chevron-down
```
Comment (mattnickles) [inline src/website/layouts/partials/sidenav.pug:20]:
```suggestion
                          i.chi-icon.icon-chevron-down.-xs
```

File: src/website/layouts/partials/sidenav.pug [hunk 2]
```diff
@@ -11,6 +11,34 @@ ul.chi-tabs.-vertical.-lg.-animated
...
+                        .chi-accordion__content.-p--0
+                          ul.chi-tabs__subtabs
+                            li(class=`${'foundations/overview' === path ? "-active" : ""}`)
```
Comment (mattnickles) [inline src/website/layouts/partials/sidenav.pug:23]: Let's change the paths for all accessibility pages to include accessibility. e.g. `${rootPath}foundations/accessibility/overview`
`${rootPath}foundations/accessibility/using-keyboard`
`${rootPath}foundations/accessibility/page-structure`

**Review by mattnickles — CHANGES_REQUESTED** [review]

File: src/website/views/foundations/overview.pug [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+ul.-text
+  li <strong>Time-based Media:</strong> Provide alternatives for time-based media.
+  li <strong>Distinguishable:</strong> Make it easier for users to see and hear content including separating foreground from background.
+  li <strong>Enough Time:</strong> Provide users enough time to read and use content.
+  li <strong>Seizures:</strong> Do not design content in a way that is known to cause seizures.
+  
+.chi-divider.-my--5
+
+p.-text
+  | CenturyLink is required to meet the Web Content Accessibility Guidelines (<a target="_blank" href="https://www.w3.org/TR/WCAG20/">WCAG 2.0</a>), Conformance
```
Comment (mattnickles) [inline src/website/views/foundations/overview.pug:59]:
```suggestion
  | Lumen is required to meet the Web Content Accessibility Guidelines (<a target="_blank" href="https://www.w3.org/TR/WCAG20/">WCAG 2.0</a>), Conformance
```

File: src/website/layouts/partials/sidenav.pug [hunk 4]
```diff
@@ -11,6 +11,34 @@ ul.chi-tabs.-vertical.-lg.-animated
                 if section.disabled
                   //- li
                   //-   a(href=`${rootPath}${section.path}`, disabled="disabled") #{section.title}
+                else if section.accessibilityAccordion
+                  li
+                    .chi-accordion.accessibility-accordion
+                      .chi-accordion__item.-expanded
+                        button.chi-accordion__trigger
+                          .chi-accordion__title #{section.title}
```
Comment (mattnickles) [inline src/website/layouts/partials/sidenav.pug:19]: Style is slightly off here in mobile. Accessibility is rendering as a Level 1 item versus Level 2 on desktop.

File: src/website/layouts/partials/sidenav.pug [hunk 5]
```diff
@@ -26,11 +54,38 @@ ul.chi-tabs.-vertical.-lg.-animated
                 if section.disabled
                   //- li
                   //-   a(href=`${rootPath}${section.path}`, disabled="disabled") #{section.title}
+                else if section.accessibilityAccordion
+                  li
+                    .chi-accordion.accessibility-accordion
+                      .chi-accordion__item
+                        button.chi-accordion__trigger
+                          .chi-accordion__title #{section.title}
+                          i.chi-icon.icon-chevron-down
```
Comment (mattnickles) [inline src/website/layouts/partials/sidenav.pug:63]:
```suggestion
                          i.chi-icon.icon-chevron-down.-xs
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f08dc008aa0e0d673b176ae713ece61d74c33d79-chi-pr.enterprise.ctl.io" target="_blank"> https://f08dc008aa0e0d673b176ae713ece61d74c33d79-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1655/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://17c2a2c06244b0642f48bdf014590c763b5ea7b9-chi-pr.enterprise.ctl.io" target="_blank"> https://17c2a2c06244b0642f48bdf014590c763b5ea7b9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1660/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cae150e0be146bca23b3e3bb2ed7438623cf7c5f-chi-pr.enterprise.ctl.io" target="_blank"> https://cae150e0be146bca23b3e3bb2ed7438623cf7c5f-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1662/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://3de8bf900ba299291147a62da4b9d783662be4b8-chi-pr.enterprise.ctl.io" target="_blank"> https://3de8bf900ba299291147a62da4b9d783662be4b8-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1664/ for more details.

**Review by mattnickles — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2dcbeba8f3628ab39e97e6ebdbb060e91bf687a7-chi-pr.enterprise.ctl.io" target="_blank"> https://2dcbeba8f3628ab39e97e6ebdbb060e91bf687a7-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1669/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-ixg7ok, https://ops.elasticbox.com/services/instances/i-05nw6i, https://ops.elasticbox.com/services/instances/i-cl8kr4, https://ops.elasticbox.com/services/instances/i-sk4wti, https://ops.elasticbox.com/services/instances/i-awuar8, https://ops.elasticbox.com/services/instances/i-sl6n0j, https://ops.elasticbox.com/services/instances/i-2rwxv7, https://ops.elasticbox.com/services/instances/i-l7djiv, https://ops.elasticbox.com/services/instances/i-d1uxzy, https://ops.elasticbox.com/services/instances/i-8vf9j8

**Pull request merged**